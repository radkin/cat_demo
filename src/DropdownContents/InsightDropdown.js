import React from "react"
import styled from "styled-components"
import {
  Icon,
  DropdownSection,
  Heading,
  HeadingLink,
  LinkList
} from "./Components"

const InsightDropdownEl = styled.div`
  width: 25rem;
`

const Flex = styled.div`
  display: flex;
  > div:first-of-type {
    margin-right: 48px;
  }
`

const InsightDropdown = () => {
  return (
    <InsightDropdownEl>
      <DropdownSection data-first-dropdown-section>
        <div>
          <Heading>Documentation</Heading>
          <p>Custom Predictive Algorithms</p>
          <Flex>
            <div>
              <h4>Get Started</h4>
              <LinkList>
                <li>
                  {/* eslint-disable-next-line */}
                  <a
                    href="#">
                    Statistics
                  </a>
                </li>
                <li>
                  {/* eslint-disable-next-line */}
                  <a
                    href="#">
                    Machine Learning
                  </a>
                </li>
                <li>
                  {/* eslint-disable-next-line */}
                  <a
                    href="#">
                    Cranial Nerve Augmentation
                    </a>
                </li>
              </LinkList>
            </div>
            <div>
              <h4>Popular Imrovements</h4>
              <LinkList>
                <li>
                  {/* eslint-disable-next-line */}
                  <a
                    href="#">
                    Cortex Nerve Density
                  </a>
                </li>
                <li>
                  {/* eslint-disable-next-line */}
                  <a
                    href="#">
                    Cortexifan Treatment
                    </a>
                </li>
                <li>
                  {/* eslint-disable-next-line */}
                  <a
                    href="#">
                    Cellular Regeneration
                  </a>
                </li>
              </LinkList>
            </div>
          </Flex>
        </div>
      </DropdownSection>
      <DropdownSection>
        <ul>
          <HeadingLink>
            {/* eslint-disable-next-line */}
            <a
              href="#">
              <Icon />NeoGenesis
            </a>
          </HeadingLink>
          <HeadingLink>
            {/* eslint-disable-next-line */}
            <a
              href="#">
              <Icon />Teleportation
            </a>
          </HeadingLink>
          <HeadingLink noMarginBottom>
            {/* eslint-disable-next-line */}
            <a
              href="#">
              <Icon />New Research
            </a>
          </HeadingLink>
        </ul>
      </DropdownSection>
    </InsightDropdownEl>
  )
}

export default InsightDropdown
