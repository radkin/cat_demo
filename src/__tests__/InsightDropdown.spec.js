import React from 'react';
import renderer from 'react-test-renderer';
import { mount } from 'enzyme';

import InsightDropdown, { Counter, dataReducer } from '../DropdownContents/InsightDropdown';

describe('InsightDropdown', () => {

  test('snapshot renders', () => {
    const component = renderer.create(<InsightDropdown />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });

});
