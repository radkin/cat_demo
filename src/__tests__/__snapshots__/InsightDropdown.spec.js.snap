// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InsightDropdown snapshot renders 1`] = `
<div
  className="sc-AxgMl bVVCIf"
>
  <div
    className="sc-AxhUy jUcgjo"
    data-first-dropdown-section={true}
  >
    <div>
      <h3
        className="sc-AxjAm ftcold"
      >
        Documentation
      </h3>
      <p>
        Custom Predictive Algorithms
      </p>
      <div
        className="sc-AxheI iQpwjd"
      >
        <div>
          <h4>
            Get Started
          </h4>
          <ul
            className="sc-AxiKw dpUUZO"
          >
            <li>
              <a
                href="#"
              >
                Statistics
              </a>
            </li>
            <li>
              <a
                href="#"
              >
                Machine Learning
              </a>
            </li>
            <li>
              <a
                href="#"
              >
                Cranial Nerve Augmentation
              </a>
            </li>
          </ul>
        </div>
        <div>
          <h4>
            Popular Imrovements
          </h4>
          <ul
            className="sc-AxiKw dpUUZO"
          >
            <li>
              <a
                href="#"
              >
                Cortex Nerve Density
              </a>
            </li>
            <li>
              <a
                href="#"
              >
                Cortexifan Treatment
              </a>
            </li>
            <li>
              <a
                href="#"
              >
                Cellular Regeneration
              </a>
            </li>
          </ul>
        </div>
      </div>
    </div>
  </div>
  <div
    className="sc-AxhUy jUcgjo"
  >
    <ul>
      <li
        className="sc-AxirZ dlCcpm"
      >
        <a
          href="#"
        >
          <div
            className="sc-AxhCb hPcABc"
          />
          NeoGenesis
        </a>
      </li>
      <li
        className="sc-AxirZ dlCcpm"
      >
        <a
          href="#"
        >
          <div
            className="sc-AxhCb hPcABc"
          />
          Teleportation
        </a>
      </li>
      <li
        className="sc-AxirZ kBSoHp"
      >
        <a
          href="#"
        >
          <div
            className="sc-AxhCb hPcABc"
          />
          New Research
        </a>
      </li>
    </ul>
  </div>
</div>
`;
